test/tests/warm_reset_during_fw_load.rs

```diff
@@ -0,0 +1,79 @@
+// Licensed under the Apache-2.0 license
+
+#![cfg(any(feature = "verilator", feature = "fpga_realtime"))]
+
+use caliptra_builder::{ImageOptions, APP_WITH_UART, FMC_WITH_UART, ROM_WITH_UART};
+use caliptra_hw_model::{mbox_write_fifo, BootParams, HwModel, InitParams, SecurityState};
+use caliptra_hw_model_types::{DeviceLifecycle, Fuses};
+use caliptra_test::swap_word_bytes_inplace;
+use openssl::sha::sha384;
+use zerocopy::AsBytes;
+
+fn bytes_to_be_words_48(buf: &[u8; 48]) -> [u32; 12] {
+    let mut result: [u32; 12] = zerocopy::transmute!(*buf);
+    swap_word_bytes_inplace(&mut result);
+    result
+}
+
+#[test]
+fn warm_reset_during_fw_load() {
```

Comment:
nit: Would probably make sense to just put this test in `warm_reset.rs`. This would also make it easier to deduplicate some of this setup code.

Reply:
Will do